assets/evaluation_on_cloud/environments/evaluations-built-in/context/evaluate_on_data.py

```diff
@@ -8,13 +8,38 @@
 import mlflow
 import pandas as pd
 import os
+import re
 import requests
 import shutil
 import sys
 
 from azure.ai.ml.identity import AzureMLOnBehalfOfCredential
 from azure.ai.evaluation import evaluate
 from save_evaluation import load_evaluator
+from model_target import ModelTarget
+
+API_VERSION = "2025-01-01-preview"
+AZURE_ENDPOINT = "AzureEndpoint"
+API_KEY = "ApiKey"
+AZURE_DEPLOYMENT = "AzureDeployment"
+TYPE = "Type"
+INPUT_EVAL_DATA_DIR = "INPUT_eval_data"
+OUTPUT_EVAL_DATA_DIR = "OUTPUT_eval_data"
```

Comment:
I think this comes from the injected env vars. 
See `parser.add_argument("--eval_output", type=str)`
No need to hardcode